### ISSUE TITLE
Add MDK quickstart user guide to docs site

### ISSUE THREAD

**Issue by sr-murthy** [OasisLMF/OasisLMF#119]
The existing MDK documentation on the docs site is poor and inadequate, and there is no user guide with examples for the different MDK commands and subcommands. A simple quickstart guide using PiWind would be an enhancement.

**Comment by mpinkerton-oasis** [comment]
I think Jupyter notebooks, probably extending the PiWind example, would be the best way to go for an introduction. More complete documentation is certainly required for the keys server data generation etc.

**Comment by sr-murthy** [comment]
I don't agree - I think we forget that the MDK is a public Python package, yet it has no appropriate quickstart guide, like almost every such package does. The normal mode of use for any Python package, especially one with a CLI, is the command line, not a Jupyter notebook.

A quickstart guide would be very simple to do, and I will be finishing it pretty quickly. Additionally, this PiWind Jupyter notebook seems to contains some bugs, as I remember some Slack conversations in the members channel where certain did not work as usual. Jupyter Notebook is an additional layer above the MDK and has nothing really to do with it.

**Comment by sr-murthy** [comment]
A useful reference and model for documentation standards - the Pandas docs and 10 minutes quickstart guide

https://pandas.pydata.org/pandas-docs/stable/10min.html
